Author: Timo Aaltonen <tjaalton@ubuntu.com>
Date:   Wed Feb 29 19:39:45 2012 +0200

    fix-script-name-cgi.patch: Use the correct name for repl-monitor perl script

--- a/admserv/cgi-src40/repl-monitor-cgi.pl.in
+++ b/admserv/cgi-src40/repl-monitor-cgi.pl.in
@@ -69,4 +69,4 @@ if ( $query->url_param('servport') ne ""
 
 # Now the real work
 print "Content-Type: text/html\n\n";
-require "@bindir@/repl-monitor.pl";
+require "@bindir@/repl-monitor";
